tests/federation_room_join_partial_state_test.go

```diff
@@ -76,12 +76,13 @@ func TestPartialStateJoin(t *testing.T) {
 		return syncToken
 	}
 
+	deployment := Deploy(t, b.BlueprintAlice)
+	defer deployment.Destroy(t)
+
 	// test that a regular /sync request made during a partial-state /send_join
 	// request blocks until the state is correctly synced.
 	t.Run("SyncBlocksDuringPartialStateJoin", func(t *testing.T) {
-		deployment := Deploy(t, b.BlueprintAlice)
-		defer deployment.Destroy(t)
-		alice := deployment.Client(t, "hs1", "@alice:hs1")
+		alice := deployment.RegisterUser(t, "hs1", "t2alice", "secret", false)
```

Comment:
`t2alice` seems duplicated, not sure if that's on purpose?

Reply:
oh silly me, I committed it on accident to test what would happen if we did that accidentally :-)